Src/Xer.Cqrs.Extensions.SimpleInjector/ICqrsEventHandlerSelector.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections.Generic;
+using System.Reflection;
+using SimpleInjector;
+
+namespace Xer.Cqrs.Extensions.SimpleInjector
+{
+    public interface ICqrsEventHandlerSelector
+    {
+        ICqrsEventHandlerSelector ByInterface(Assembly assembly);
+        ICqrsEventHandlerSelector ByInterface(Assembly assembly, Lifestyle lifestyle);
+        ICqrsEventHandlerSelector ByInterface(IEnumerable<Assembly> assemblies);
```

Comment:
Why not use the `params Assembly[] assemblies` syntax?

Reply:
I needed to pass in a Lifestyle as 2nd parameter to set lifestyle of handlers. I had some problems using params due to method overload resolution.

ByInterface(params Assembly[] assemblies) // Default to Lifestyle.Transient
ByInterface(Assembly assembly, Lifestyle lifestyle) // Set lifestyle
ByInterface(Lifestyle lifestyle, params Assembly[] assemblies) // Lifestyle is first parameter?

Reply:
Last solutions sound good to me pass the lifetime as the first param.